infra: edge zones (#296)
apache/cloudstack-documentation: source/installguide/configuration.rst

Advanced Zone Configuration
~~~~~~~~~~~~~~~~~~~~~~~~~~~

> For Advanced zone, you may chose Edge which will allow creating an Edge. If Edge is not selected then wizard will allow creating a Core zone.

Reviewer: can you explain or modify this sentence? this is a checkbox ? maybe you mean 
```suggestion
For Advanced zone, you may chose to select Edge which will allow creating an Edge Zone. If Edge is not selected then wizard will continue creating a Core zone.
```

#. Click Launch.

Edge Zone
*********

.. note::

>    Support for Edge zone has been added with 4.18.0 and these zones will only support KVM hypervisor

Reviewer: ```suggestion
   Support for Edge zones has been added with 4.18.0 and these zones will only be supported on KVM hypervisors
```

> Edge zone is a simpler, light-weight zone which may often contain a single hypervisor host. There will be no need for shared storage, public and management physical networks for an Edge zone.

Reviewer: ```suggestion
An Edge Zone is a simpler, light-weight zone which may often contain a single hypervisor host. There will be no need for shared storage, public and management physical networks for an Edge zone.
```

> To work with limited compute resources, an Edge zone will not deploy system VMs. This type of zone only support shared and L2 guest networks. For virtual routers of a shared guest network, a direct-download System VM must be added after adding the zone.

Reviewer: ```suggestion
To work with limited compute resources, an Edge zone will not deploy system VMs. This type of zone only supports shared and L2 guest networks. For virtual routers of a shared guest network, a direct-download System VM must be added after adding the zone.
```
